Jellyfin.Server/Migrations/Routines/DisableZealousLogging.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.IO;
+using MediaBrowser.Common.Configuration;
+using MediaBrowser.Model.Configuration;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Serilog;
+using ILogger = Microsoft.Extensions.Logging.ILogger;
+
+namespace Jellyfin.Server.Migrations.Routines
+{
+    /// <summary>
+    /// Updater that takes care of bringing configuration up to 10.5.0 standards.
```

Comment:
@mark-monteiro this should be explained better in your pull request.

Reply:
Yes, I've described the changes in detail there: https://github.com/jellyfin/jellyfin/pull/2535/files#diff-5559799524e9efb63701fb597cf293b9